src/App.js

```diff
@@ -1,16 +1,33 @@
-import React from 'react';
+import React from "react";
+
+let todoList = [
+  {
+    id: 1234,
+    title: "Complete CTD assignment",
+  },
+  {
+    id: 4321,
+    title: "Go grocery shopping",
+  },
+  {
+    id: 9876,
+    title: "Go the gym",
+  },
+  {
+    id: 4987,
+    title: "Do laundry",
+  },
+];
 
 function App() {
   return (
-    <div style={{ textAlign: 'center' }}>
-      <header>
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
+    <div>
+      <h1>Todo List</h1>
+      <ul>
+        {todoList.map((item) => {
+          return <li key={item.id}>{item.title}</li>;
+        })}
+      </ul>
     </div>
```

Comment:
This all looks great! One thing to note (and I'm not sure if it's been covered in your curriculum so far) is that if you need to group multiple top-level elements together you can wrap them in a [React fragment](https://react.dev/reference/react/Fragment) rather than a `div` or whatever. A React fragment looks like this `<>...</>` so in practice your code would look like this:

```suggestion
    <>
      <h1>Todo List</h1>
      <ul>
        {todoList.map((item) => {
          return <li key={item.id}>{item.title}</li>;
        })}
      </ul>
    </>
```